recipes/libtheora/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: libtheora
+  version: 1.1.1
+
+source:
+  fn: libtheora-1.1.1.tar.bz2
+  url: http://downloads.xiph.org/releases/theora/libtheora-1.1.1.tar.bz2
+  md5: 292ab65cedd5021d6b7ddd117e07cd8e
+
+build:
+    number: 0
+    skip: True  # [not linux]
+
+requirements:
+  build:
+    - pkg-config
+    - toolchain
+    - libogg
+    - libvorbis
+    - libpng <1.6
```

Comment:
There was a bug for newer versions at some point, not sure if this was fixed already.